### ISSUE TITLE
Importing plugin via ES6: TypeError: _videoJs2.default is undefined

### ISSUE THREAD

**Issue by brookslyrette** [videojs/generator-videojs-plugin#70]
I'm trying to use `import` to add a plugin. The plugin was created using this generator. 

I'm doing this in my app:
```
import videojs from 'video.js';
import 'videojs-plugin-test';
```

When the plugin starts up it errors due to: 
`TypeError: _videoJs2.default is undefined`

I can see this is part of the generated output of the built plugin:

```
// Register the plugin with video.js.
_videoJs2['default'].plugin('pluginTest', pluginTest);
```
Am I missing a step?

**Comment by gkatsev** [comment]
Are you using babel to compile down your imports?

**Comment by brookslyrette** [comment]
We are via babelify/browserify

**Comment by gkatsev** [comment]
I wonder if the issue is related to https://github.com/videojs/generator-videojs-plugin/pull/69
I'm using browserify and babelify [elsewhere](https://github.com/videojs/videojs.com/pull/51/files#diff-03294417c3330c12bcd42b405d1b1af6R1) and it's working. Granted I had to use updated versions of plugins like videojs-playlist (https://github.com/brightcove/videojs-playlist/pull/44)

**Comment by brookslyrette** [comment]
You nailed it. Removing "browserify-shim" from the transforms list and adding it to the build:js:browserify command seems to have fixed ES6 imports.

Added:
```
"scripts": {
    ...
    "build:js:browserify": "browserify . -s videojs-plugin-test -t browserify-shim -o dist/videojs-plugin-test.js",
    ....
},
...
"browserify-shim": {
    "qunit": "global:QUnit",
    "sinon": "global:sinon",
    "video.js": "global:videojs"
},
```

Removed: 
```
"browserify": {
   "transform": [
     "browserify-shim",
     "browserify-versionify"
  ]
},
```

Any insight as to why this fixes the import issue? Doesn't `-t browserify-shim` run the same transform as before?

**Comment by gkatsev** [comment]
It's because without #69, browsery-shim is being applied when you run browserify yourself, meaning that the plugins expect videojs to be available globally, instead of just being required/imported like normal.
The one thing that prevented #69 from being merged in is that we weren't sure how browserify handles being given transforms both in the config *and* the command line.

**Comment by BrandonOCasey** [comment]
should be fixed now that #69 is merged
